app/views/users/_fetch_user_info.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="chip">
```

Comment:
The role passed when this partial is called is not been displayed here. Why is that?